### PR TITLE
SNOW-1859140: Skip Cortex apply tests for Jenkins

### PR DESCRIPTION
**Pull request by sfc-gh-lmukhopadhyay**
1. Which Jira issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   

   Fixes SNOW-1859140

2. Fill out the following pre-review checklist:

   - [x] I am adding a new automated test(s) to verify correctness of my new code
      - [ ] If this test skips Local Testing mode, I'm requesting review from @snowflakedb/local-testing
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency
   - [ ] If this is a new feature/behavior, I'm adding the Local Testing parity changes.
   - [x] I acknowledge that I have ensured my changes to be thread-safe. Follow the link for more information: [Thread-safe Developer Guidelines](https://github.com/snowflakedb/snowpark-python/blob/main/CONTRIBUTING.md#thread-safe-development)

3. Please describe how your code solves the related issue.

   Skip Cortex function apply tests when run on Jenkins

### CODE REVIEW

File: tests/integ/modin/test_apply_snowpark_python_functions.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def test_apply_snowpark_python_function_not_implemented():
         pd.DataFrame({"a": [1, 2, 3]}).apply(asc, args=(1, 2))
 
 
+@pytest.mark.skip("SNOW-1859087 snowflake.cortex.summarize SSL error")
```
Comment (sfc-gh-azhan) [inline tests/integ/modin/test_apply_snowpark_python_functions.py:74]: Can we only skip it when it is run by jenkins?

Comment (sfc-gh-lmukhopadhyay) [reply tests/integ/modin/test_apply_snowpark_python_functions.py]: Yes, I added that now

**Review by sfc-gh-azhan — APPROVED** [review]

File: tests/integ/modin/test_apply_snowpark_python_functions.py (hunk 1 above)
Comment (sfc-gh-helmeleegy) [reply tests/integ/modin/test_apply_snowpark_python_functions.py]: I updated the PR title accordingly.

**Review by sfc-gh-helmeleegy — APPROVED** [review]
Looks good. Thanks, Labanya.

**Review by sfc-gh-yuwang — APPROVED** [review]

**Pull request merged**